.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
-before_install: npm i -g npm@1.4.28
```

Comment:
This version of NPM is problematic for two reasons:

- It doesn't work on newer versions of Node: https://travis-ci.org/auth0/node-wsfed/jobs/346731487
- It doesn't support scoped packages like `@auth0/thumbprint`.